qiskit/transpiler/passes/utils/gate_direction.py

```diff
@@ -277,11 +285,21 @@ def _run_target(self, dag, wire_map):
                         f"The circuit requires a connection between physical qubits {qargs}"
                         f" for {node.name}"
                     )
-            else:
+            elif self.target.instruction_supported(node.name, qargs):
+                continue
+            elif self.target.instruction_supported(node.name, swapped) or dag.has_calibration_for(
+                _swap_node_qargs(node)
+            ):
                 raise TranspilerError(
                     f"Flipping of gate direction is only supported "
-                    f"for {list(self._static_replacements)} at this time, not '{node.name}'."
+                    f"for {list(self._KNOWN_REPLACEMENTS)} at this time, not '{node.name}'."
```

Comment:
It's worth also mentioning only opposite direction is defined?

Reply:
Pulse level user might be lazy for managing order matching of virtual and logical qubits. So one might hit this error with some typo. Saying having calibration of another direction might help the user to find what's wrong in his code. Otherwise one might start investigating how to implement gate flipping.

Reply:
How is the new message in 607914ea6 for you?

Reply:
Thanks. I like new one.